src/main/java/org/reaktivity/nukleus/tcp/internal/stream/ClientStreamFactory.java

```diff
@@ -359,10 +360,9 @@ public void doConnect(
                 poller.doRegister(channel, OP_CONNECT, request);
             }
         }
-        catch (IOException ex)
+        catch (UnresolvedAddressException | IOException ex)
```

Comment:
This handles the case of `channel.connect(...)` throwing an exception on stack, but what if the same exception is deferred until later when `OP_CONNECT` fires and `channel.finishConnect()` throws the exception then instead?